mhs-reference-implementation/selenium_tests/page_objects/methods.py

```diff
@@ -1,19 +1,20 @@
 import os
-from pathlib import Path
 
-from definitions import ROOT_DIR
 from selenium_tests.page_objects import interactions, xml_parser
 
 HUMAN_READABLE = 'Payload stuff'
-DATA_PATH = 'selenium_tests/data'
 
 
 def get_asid():
-    # The asid should be stored in data\local_asid file (which is excluded from GIT
-    with (Path(ROOT_DIR) / DATA_PATH / "local_asid").open() as asid_file:
-        asid = asid_file.readline()
+    # The asid should be set in the 'Environment variables' section of the Run/Debug Configurations
+    # ...if this is not set, it will default to '1234567890', which will cause test failures!)
+    return os.environ.get('ASID', os.environ.get('ASID', 1234567890))
 
-    return asid
+
+def get_hostname():
+    # The hostname should be set in the 'Environment variables' section of the Run/Debug Configurations
+    # ...if this is not set, it will default to 'http://localhost/'
+    return "http://" + os.environ.get('MHS_ADDRESS', 'localhost') + "/"
```

Comment:
Can I suggest we record the details (and maybe an example for pycharm) about which environment variables we need and the format they should take (so mhs address should be a 'bare' url with no http prefix or '/' postfix) in a README for the integration tests. Just a bit easier to find than in the code